docs/components/table-view.md

```diff
@@ -699,6 +699,10 @@ interface TablePaginationAttributes {
 
 A boolean to hide pagination element (defaults to `false`).
 
+### hidePaginationWhenOptional
+
+Set this to `true` to hide pagination when the table record count is less than or equal to the page size. Defaults to `true`.
```

Comment:
Shall we flip the logic and use something like `autoHidePagination` so that it defaults to `false`?

Reply:
I suggest we don't in order to keep component API as close to original KTable API as possible.

Reply:
FWIW this is a new component, there is no requirement to follow the old component APIs 1:1 as long as the changes are documented correctly and picked up in Konnect accordingly.

Reply:
Okay, even if so, `autoHidePagination` sounds confusing to me. `hidePaginationWhenOptional` is a prop name our engineers are familiar with from KTable and KCatalog, it's intuitive.

Reply:
What I mainly want to say is that we should strive to set the default value of `boolean` props to false, but the specific name in this case is of course open for discussion.

It seems that the `<KTable>` component's `hidePaginationWhenOptional` prop [defaults to `false`](https://github.com/Kong/kongponents/blob/be5df2d4467087add361cd0564110bafde52dd4d/src/components/KTable/KTable.vue#L494), are we changing its default value in the new tables?

Reply:
Yes, it's `true` in KTableView but `false` in KTableData.

Reply:
We point that out in docs here: https://deploy-preview-2405--kongponents.netlify.app/components/table-data.html#base-ktableview-component. Is that sufficient or would it be better if the default value for this prop was matching across the two components (in that case I would suggest it defaults to `false`)?

Reply:
Pushed a commit to change `hidePaginationWhenOptional` default value to `false` in KTableView.